app/assets/stylesheets/templates/_admin.scss

```diff
@@ -0,0 +1,44 @@
+.admin {
+
+  div.header {
+
+    * {
+      margin: 0;
+      margin-bottom: 1.0rem;
+    }
+
+    h1 {
+      margin-bottom: 1.0rem;
+    }
```

Comment:
These look like redundant rules. keep in mind that we do have spacing classes that you can add to html.